src/rt/cpp/cown.h

```diff
@@ -83,7 +90,7 @@ namespace verona::cpp
     /**
      * Accesses the internal Verona runtime cown for this handle.
      */
-    Cown* underlying_cown()
+    ActualCown<T>* underlying_cown()

```

Comment:
Can we have a comment to potentially revert this if a better interface comes along for the inactive cown in the future?